### PR TITLE
Ansible-lint checks

### PR DESCRIPTION
**Pull request by ra-rau**
- Add attribute position check
- Add check of the usage of the ansible-fqcn
- partly osism/issues#196
- partly osism/issues#178

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: .ansible-lint-rules/osism_fqcn_list.yaml [hunk 1]
```diff
@@ -0,0 +1,474 @@
+---
+builtin:
```
[the remark this answers is not in the document]
Comment (ra-rau) [reply .ansible-lint-rules/osism_fqcn_list.yaml:2]: Done

File: .ansible-lint-rules/osism_fqcn_list.yaml [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+  - openstack.cloud.volume_snapshot
+  - openstack.cloud.volume_snapshot_info
+  - openstack.cloud.openstack
+
+other:
+  - scan_services
+  - hashivault_secret
+  - hashivault_secret_engine
+  - hashivault_auth_method
+  - hashivault_policy
```
[the remark this answers is not in the document]
Comment (ra-rau) [reply .ansible-lint-rules/osism_fqcn_list.yaml:474]: The other are only there for the moment. Them will be fixed later if we can change them.

File: .ansible-lint-rules/osism_attribute_order.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        counter_prev = 0
+        for attribute in osism_attribute_order_list["osism_attribute_order_list"]:
+            if attribute in raw_task:
+                attribute_list = [*raw_task]
+                counter = attribute_list.index(attribute)
+                if counter < counter_prev:
+                    return f"{attribute_list[counter_prev]} is not at the right place"
+                else: 
+                    counter_prev = counter
+        return False
```
[the remark this answers is not in the document]
Comment (ra-rau) [reply .ansible-lint-rules/osism_attribute_order.py:43]: Done

File: .ansible-lint-rules/osism_attribute_order.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        raw_task = task["__raw_task__"]
+        counter = 0
+        counter_prev = 0
+        for attribute in osism_attribute_order_list["osism_attribute_order_list"]:
+            if attribute in raw_task:
+                attribute_list = [*raw_task]
+                counter = attribute_list.index(attribute)
+                if counter < counter_prev:
+                    return f"{attribute_list[counter_prev]} is not at the right place"
+                else:
```
[the remark this answers is not in the document]
Comment (ra-rau) [reply .ansible-lint-rules/osism_attribute_order.py:41]: Done

**Review by osfrickler — APPROVED** [review]

**Pull request merged**